packages/process/_test.pony

```diff
@@ -366,6 +369,91 @@ class iso _TestStdinWriteBuf is UnitTest
     end
     h.complete(false)
 
+class _TestChdir is UnitTest
+  fun name(): String =>
+    "process/Chdir"
+
+  fun exclusion_group(): String =>
+    "process-monitor"
+
+  fun ref apply(h: TestHelper) =>
+    let parent = Path.dir(Path.cwd())
+    // expect path length + \n
+    let notifier: ProcessNotify iso = _ProcessClient(parent.size() + 1, "", 0, h)
+    try
+      let auth = h.env.root as AmbientAuth
+
+      let path = FilePath(auth, "/bin/pwd")?
```

Comment:
This will not work on windows. Appveyor is currently running smoothly because process tests are still excluded on windows/appveyor. Maybe @cquinn has got an idea how to make those tests more portable.